### PR TITLE
Add make dbshell cmd to access database in cli

### PR DESCRIPTION
**Pull request by rlecellier**
## Proposal

Add Makefile entry to acces postgresql docker in cli using: `make psql`

### CODE REVIEW

**Comment by rlecellier** [comment]
Everything work's well here on richie, but i still [have a error on joanie](https://github.com/openfun/joanie/pull/203) where the changes is exactly the same :thinking: 

```sh
$ make dbshell
docker-compose exec app python manage.py dbshell
CommandError: You appear not to have the 'psql' program installed or on your path.
make: *** [Makefile:176 : dbshell] Erreur 1
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -756,8 +760,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Hide unpublished pages from public version of organizations list page
 - Improve sanitizing of input text on CKEditor plugin
-- Replace course-detail__run-cta in fragment_course_run template by
-  course-run-enrollment__cta
+- Replace course-detail**run-cta in fragment_course_run template by
+  course-run-enrollment**cta
```
Comment (sampaccoud) [inline CHANGELOG.md:764]: oups. You have to hold your linter tighter! Maybe remove all reformatting from your commit and eventually add a separated commit for the legitimate ones?

File: Dockerfile [hunk 2]
```diff
@@ -95,6 +95,12 @@ USER root:root
 
 WORKDIR /app
 
+# Install psql and mysql
+RUN apt-get update && \
+    apt-get install -y postgresql-client && \
+    apt-get install -y default-mysql-client && \
```
Comment (sampaccoud) [inline Dockerfile:101]: you should be able to write it in one line ?
```suggestion
    apt-get install -y postgresql-client default-mysql-client && \
```

File: Makefile [hunk 3]
```diff
@@ -323,6 +323,13 @@ i18n-generate-front: build-ts
 	@$(YARN) extract-translations
 .PHONY: i18n-generate-front
 
+# -- Database
+# -- docker exec -it richie_postgresql_1 bash -c "psql -U fun richie"
+
```
Comment (sampaccoud) [inline Makefile:328]: should we commit this?

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**